signal_your_workflow/external_signal_main.py

```diff
@@ -0,0 +1,26 @@
+import asyncio
+
+from temporalio.client import Client
+from signal_external_wf_dacx import WorkflowA, WorkflowB
+
+
+async def main():
+    client = await Client.connect("localhost:7233")
+    handle_b = await client.start_workflow(
+        WorkflowB.run,
+        id="workflow-b",
+        task_queue="signal-tq",
+    )
+    await handle_b.signal(WorkflowA.your_signal, "workflow-a")
```

Comment:
You're referencing a signal on another workflow than the handle is on here. I was hoping MyPy would be smart enough to catch this. Regardless, there is no `your_signal` on `handle_b`.